conan/cli/commands/__init__.py

```diff
@@ -30,9 +32,13 @@ def default(self, o):
         return JSONEncoder.default(self, o)
 
 
-def json_formatter(data):
+def default_json_formatter(data):
     myjson = json.dumps(data, indent=4, cls=ConanJSONEncoder)
-    return myjson
+    ConanOutput(stream=sys.stdout).writeln(myjson)
```

Comment:
I am not sure about this. We were discussing about how ``ConanOutput`` was always for our messages to stderr, while we would use other things for stdout. Becaye also there is extra configurations, like loggers or the logger-as-json that can affect how the ``ConanOutput`` behaves, and that would be bad for the ``stdout`` output, wouldn't it?

Reply:
Ok, I see, the probably what needs to be used is `cli_out_write` (we can change the name later perhaps), I'll make the changes and we see from there